pilz_testutils/include/pilz_testutils/async_test.h

```diff
@@ -73,20 +74,24 @@ class AsyncTest
     void triggerClearEvent(const std::string &event);
 
     /**
-     * @brief Will block until the event given by clear_event is triggered. Unblocks immediately, if the event was
-     * triggered in advance.
+     * @brief Will block until the event given by clear_event is triggered or a timeout is reached.
+     * Unblocks immediately, if the event was triggered in advance.
```

Comment:
```suggestion
     * Unblocks immediately, if the event was triggered before the timeout.
```
I would prefer this wording. `in advance` sounds to me like being triggered before the barricade is defined. Which of cause makes no sense. But I think it is clearer like this

Reply:
> sounds to me like being triggered before the barricade is defined

That is exactly what it means.

Reply:
However I tried to come up with a better explanation.

Reply:
yes, thats good!